mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -134,7 +134,8 @@ def gridded_inverse_fourier_transform_stack(kspace_data_sorted,
             kspace_data_sorted[i*stack_len:(i+1)*stack_len],
             grid,
             method=method,
-            fill_value=0)
+            fill_value=0
```

Comment:
I think that what @zaccharieramzi  asked is to add a comma after `fill_value=0`, otherwise there is no point of doing this